src/services/projects.js

```diff
@@ -634,22 +635,22 @@ module.exports = class ProjectsHelper {
 					excluded_user_ids: [userDetails.id],
 				})
 
-				//return error message if the reviewer is invalid or not found
-				if (reviewers.length <= 0 || bodyData.reviewer_ids > reviewers.length) {
-					return responses.failureResponse({
-						message: 'REVIEWER_IDS_NOT_FOUND',
-						statusCode: httpStatusCode.bad_request,
-						responseCode: 'CLIENT_ERROR',
-					})
-				}
+				if (!reviewers.success) throw new Error('REVIEWER_IDS_NOT_FOUND')
 
 				let reviewerIds = []
 
 				//written as a backup will remove once the user service PR merged
 				if (Array.isArray(reviewers?.data?.result?.data) && reviewers.data.result.data.length > 0) {
 					reviewerIds = reviewers.data.result.data.map((item) => item.id)
+				} else {
+					// If no valid reviewers data is found, return an error response
+					throw new Error('REVIEWER_IDS_NOT_FOUND')
 				}
 
+				//return error message if the reviewer is invalid or not found
+				if (bodyData.reviewer_ids.length >= reviewers.data.result.data.length)
```

Comment:
This have an edge case,  
since bodyData can be passed from the frontend , they can repeat the reviewer_ids. but while searching we get only unique values. So in such a case we cannot go for length match. 
Ex : 
in body if we receive 
[1,2,2,3,4,5,5] length will be 7
and the result from user service will be 5.

Try this or any other feasible method 
const areArraysEqual = _.isEqual(_.sortBy(array1), _.sortBy(array2));

@priyanka-TL